src/borg/archiver.py

```diff
@@ -2384,6 +2403,9 @@ def process_epilog(epilog):
         subparser.add_argument('--force', dest='forced',
                                action='store_true', default=False,
                                help='force deletion of corrupted archives')
+        subparser.add_argument('--zap', dest='zap',
+                               action='store_true', default=False,
+                               help='force manifest removal of corrupted archives')
```

Comment:
Hm, ok. I see. Yes. Wouldn't be straightforward to detect. Perhaps this should be more like a "--force --force" option, so that it's clear that this should only be used if it's too corrupt to work with even "--force"?